lib/grpc/server/supervisor.ex

```diff
@@ -48,7 +48,7 @@ defmodule GRPC.Server.Supervisor do
     check_deps_version()
 
     children =
-      if Application.get_env(:grpc, :start_server, false) do
+      if opts[:start_server] or (not Keyword.has_key?(opts, :start_server) and Application.get_env(:grpc, :start_server) do
```

Comment:
End parenthesis is missing? 

```suggestion
      if opts[:start_server] or (not Keyword.has_key?(opts, :start_server) and Application.get_env(:grpc, :start_server)) do
```